auto_nag/bzcleaner.py

```diff
@@ -219,6 +234,70 @@ def add_auto_ni(self, bugid, data):
             }
         return True
 
+    def add_prioritized_action(
+        self, bug, quota_name, needinfo=None, autofix=None, allow_multi_ni=False
```

Comment:
Let's remove the `allow_multi_ni` parameter and assume it is False, this way we reduce complexity. we can add it in case a tool needs it in the future.

Reply:
Same for quota_name, we can assume we have only one kind of action per tool and just have a single queue. We can add support for multiple queues later in case we need it

Reply:
> Let's remove the `allow_multi_ni` parameter and assume it is False, this way we reduce complexity. we can add it in case a tool needs it in the future.

Done in 190cbfb84ccd7e8bc2fa2d6c2520449655807093

Reply:
> Same for quota_name, we can assume we have only one kind of action per tool and just have a single queue. We can add support for multiple queues later in case we need it

It is used to count actions and ni for triage owners, see:
https://github.com/mozilla/relman-auto-nag/blob/c92e238ab5e677fa807a74438a540369167f032d/auto_nag/scripts/assignee_no_login.py#L90

The name is generic and not called `triage_owner` because we could apply the limit based on anything; it could be the team, component, or even the limit for the whole tool at that run. I added a comment in f5ab4bde4b595f351f892573ef3e4ae3a90492e4.